Unit2/Unit2-Project/README.md

```diff
@@ -0,0 +1,41 @@
+# Manage and Deploy a Project with Git
+##### Credit to OpenClassrooms
```

Comment:
Quick clarification here to make sure we're on the same page with our general approach. I was thinking we'd only do READMEs for the Unit projects, and for the Mini-Project we would just link to the OpenClassrooms (or other resources) and have the students follow that. Perhaps the Mini-Project README would just have a link plus any instructions on how to turn in the assignment. I think we should avoid duplicating content both because it might be a potential copyright violation and also to not cause potential confusion for students.

It seems like this is the approach you are taking but just want to double-check